share/completions/sv.fish

```diff
@@ -0,0 +1,80 @@
+# Completions for sv
+# A part of the runit init system
+# Author: Leonardo da Rosa Eugênio <lelgenio@disroot.org>
+
+
+set -l commands \
+    status up down once s u d o pause cont hup \
+    alarm interrupt quit 1 2 term kill exit p c h \
+    a i q 1 2 t k e start stop reload restart \
+    shutdown force-stop force-reload force-restart force-shutdown \
+    try-restart check
+
+
+function __fish_complete_sv_list_services
+    for dir in "$SVDIR" /run/runit/service/ /etc/runit/current /etc/runit/runsvdir/
+        test -d "$dir"
```

Comment:
no need to quote any of $dir, $service or $services[1] in fish, because they are all lists of exactly one element
probably the same for $SVDIR